### ISSUE TITLE
Arrowheads on dashed lines are rendered wrong by Safari

### ISSUE THREAD

**Issue by dtolnay** [ivanceras/svgbob#69]
As of current master:

```console
$ cargo build
$ echo ' <~~~~~~~~~~~~~~>' | target/debug/svgbob > test.svg
```

It looks okay in Chrome and Firefox, but Safari does this:

<img width="410" alt="Screen Shot 2020-12-06 at 12 27 40 PM" src="https://user-images.githubusercontent.com/1940490/101291546-70294980-37be-11eb-95a7-0c24b4abc014.png">

It would be good to avoid `marker-start`/`marker-end` and instead use a `polygon class="filled"` to position and draw the arrowheads.

**Comment by ivanceras** [comment]
Initially, I wanted the `start-marker` and `end-marker` since it creates a coherent drawing element in a sense that if you further enhance the resulting `svg` in an svg editor such as inkscape, the user can freely manipulate the lines, such as rotating the lines, dragging one of the endpoints. Rendering these arrow heads as polygon may simplify the algorithms in the code, but also losses the nice features of creating a more coherent output. 

Might be a good alternative to do it, to keep a consistent look across browsers. This could also be implemented as a flag in the Settings.

**Comment by ivanceras** [comment]
This [commit](https://github.com/ivanceras/svgbob/commit/6258b919e17274975cb115139cce81796d10c7fd) adds a parameter in the Settings to specify whether or not Arrow heads will merge with lines. The default value is false, hence the arrow heads will be rendered as polygons instead of a marker to a line. A published version 0.5.0 of the crate have this change.
